db/re/skill_db.yml

```diff
@@ -721,27 +721,27 @@ Body:
     AfterCastActDelay: 500
     Duration2:
       - Level: 1
-        Time: 2500
+        Time: 3000
       - Level: 2
-        Time: 5500
+        Time: 6000
       - Level: 3
-        Time: 8500
+        Time: 9000
       - Level: 4
-        Time: 11500
+        Time: 12000
       - Level: 5
-        Time: 14500
+        Time: 15000
       - Level: 6
-        Time: 17500
+        Time: 18000
       - Level: 7
-        Time: 20500
+        Time: 21000
       - Level: 8
-        Time: 23500
+        Time: 24000
       - Level: 9
-        Time: 26500
+        Time: 27000
       - Level: 10
-        Time: 29500
+        Time: 30000
       - Level: 11
-        Time: 32500
+        Time: 33000
```

Comment:
Should be 30000. (This is special boss skill marker for us but it should have the same time as level 10.)